### ISSUE TITLE
Missing validation of child parameter child properties

### ISSUE THREAD

**Issue by savage-alex** [stoplightio/spectral#1400]
Spectral VScode version 0.2.4 and spectral 5.6.0

Is not flagging an issue when a designer forgets to include a schema for a path parameter and enters type directly:

```yaml
/required-actions/{actionId}:
  put:
    description: 'The description'
    operationId: updateRequiredActionById
    tags:
      - Required-Actions
    parameters:
      - name: alias
        in: path
        type: string
        required: true
        example: 'VERIFY_EMAIL'
      - name: default
        in: path
        type: boolean
        required: true
        example: false 
```

**To Reproduce**

Given the above OAS, lint the definition and no error will be raised

**Expected behaviour**
type is flagged as an invalid additional property

**Additional context**
SwaggerHub is now able to notice this but spectral was not :-(

**Comment by P0lip** [comment]
Hey!
I'm sorry for keeping you waiting. Happy to inform that the PR addressing the issue has just been opened.
